dwh/query_lib_spark.py

```diff
@@ -0,0 +1,486 @@
+# Copyright 2021 Google LLC
```

Comment:
I have not reviewed the content of this file as I assumed it is mostly a copy from the old version. Please let me know if that is not the case (also again it would be nice if we can convince git to mark this as a copy to better highlight the diffs).

Reply:
Okay, this sounds good; thanks.